polygon/sync/store.go

```diff
@@ -52,13 +52,18 @@ type bridgeStore interface {
 	ReplayInitialBlock(ctx context.Context, block *types.Block) error
 }
 
-func NewStore(logger log.Logger, executionStore executionStore, bridgeStore bridgeStore) *ExecutionClientStore {
+type wiggleCalculator interface {
```

Comment:
I would suggest moving this interface to`wiggle.go`

Reply:
Interfaces should be declared at the consumer level.